src/components/dictionaryConcepts/containers/DictionaryConcepts.jsx

```diff
@@ -137,26 +135,24 @@ export class DictionaryConcepts extends Component {
   }
 
   fetchNextConcepts() {
-    this.props.paginateConcepts(null, this.state.conceptLimit + 10, this.state.conceptOffset + 10);
+    this.props.paginateConcepts(null, this.state.conceptLimit + 20, this.state.conceptOffset + 20);
     this.setState(state => ({
-      conceptOffset: state.conceptOffset + 10,
-      conceptLimit: state.conceptLimit + 10,
+      conceptOffset: state.conceptOffset + 20,
+      conceptLimit: state.conceptLimit + 20,
     }));
   }
   fetchPrevConcepts() {
-    this.props.paginateConcepts(null, this.state.conceptLimit - 10, this.state.conceptOffset - 10);
+    this.props.paginateConcepts(null, this.state.conceptLimit - 20, this.state.conceptOffset - 20);
     this.setState(state => ({
-      conceptOffset: state.conceptOffset - 10,
-      conceptLimit: state.conceptLimit - 10,
+      conceptOffset: state.conceptOffset - 20,
+      conceptLimit: state.conceptLimit - 20,
     }));
   }
 
   checkMembershipStatus(username) {
     const {
       match: {
-        params: {
-          type, typeName,
-        },
+        params: { type, typeName },
```

Comment:
Is this a formatting change?

Reply:
@dondrzzy yes, It's formatting change.

Reply:
Those should be best be handled in a different PR.